src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,78 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React from 'react';
+
+type Props = {
+  inputState: string
+  setInputState: (inputState: string) => void
+  setToggleFilter: (toggleFilter: boolean | null) => void
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  inputState, setInputState, setToggleFilter,
+}) => {
+  const getToggleValue = (value: string) => {
+    switch (value) {
```

Comment:
you can do switch case right into filter in App file

Reply:
if you will do like i suggested, then toggleValue and toggleFilter will unit, because they refer to the same logic